code/game/gamemodes/scoreboard.dm

```diff
@@ -33,76 +59,70 @@
 	for(var/i in total_antagonists)
 		log_game("[i]s[total_antagonists[i]].")
 
-	// Score Calculation and Display
+	//Score Calculation and Display
 
-	// Who is alive/dead, who escaped
-	for (var/mob/living/silicon/ai/I in mob_list)
-		if (I.stat == 2 && I.z == 1)
-			score["deadaipenalty"] = 1
-			score["deadcrew"] += 1
+	//Run through humans for diseases, also the Clown
+	for(var/mob/living/carbon/human/I in mob_list)
+		if(I.viruses) //Do this guy have any viruses ?
```

Comment:
I attempted this and it did not compile, I'm not certain if this proc outputs correctly